audit-logs/chart/templates/logs-collector.yaml

```
@@ -0,0 +1,386 @@
+{{/*
+SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+SPDX-License-Identifier: Apache-2.0
+*/}}
+
+{{- if .Values.openTelemetry.logsCollector.enabled }}
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: logs
```

The same name? Or will there be a dedicated `kind: OpenTelemetryCollector` specification?
```suggestion
  name: audit-logs
```